.github/workflows/continuous-delivery.yaml

```diff
@@ -7,7 +7,6 @@ on:
 
 jobs:
   release:
-    if: contains(github.event.commits[0].message, 'chore(release)') == false
```

Comment:
Did you test this? The last time that I did, releases started looping...
It’s why I added it.

Reply:
I'll stop it if it does, but it should never do that.